README.md

```diff
@@ -387,4 +388,4 @@ or
 **Copy the addon to the Kodi addon directory on Mac**
 
 1. `rm -rf "$HOME/Library/Application Support/Kodi/addons/pvr.iptvsimple"`
-2. `cp -rf $HOME/xbmc-addon/addons/pvr.iptvsimple "$HOME/Library/Application Support/Kodi/addons"`
\ No newline at end of file
+2. `cp -rf $HOME/xbmc-addon/addons/pvr.iptvsimple "$HOME/Library/Application Support/Kodi/addons"`
```

Comment:
What's the change here?

Reply:
Used a line for the change and all was shifted by one line.